docker/Dockerfile.backend

```diff
@@ -15,6 +15,12 @@ RUN apt-get update -y && apt-get upgrade -y \
     python3-matplotlib \
     unzip \
     wget
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get install -y nodejs
+
+RUN curl -LO https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Comment:
Is it possible to pin the version?

Reply:
not required. IIRC, it is for the FE tests that run the tests on chrome haedless and surely it is not related to the e2e tests issue we are currently facing as it is installing google chrome within the container.